doc/lua_api.txt

```diff
@@ -849,24 +849,42 @@ A positional sound will only be heard by players that are within
 `SimpleSoundSpec`
 -----------------
 
-* e.g. `""`
-* e.g. `"default_place_node"`
-* e.g. `{}`
-* e.g. `{name = "default_place_node"}`
-* e.g. `{name = "default_place_node", gain = 1.0}`
-* e.g. `{name = "default_place_node", gain = 1.0, pitch = 1.0}`
+Specifies a sound name, gain (=volume) and pitch.
+This is either a string or a table.
 
+In string form, you just specify the sound name or
+the empty string for no sound.
+
+Table form has the following fields:
+
+* `name`: Sound name
+* `gain`: Volume (`1.0` = 100%)
+* `pitch`: Pitch (`1.0` = 100%)
+
+`gain` and `pitch` are optional and default to `1.0`.
+
+Examples:
+
+* `""`: No sound
+* `{}`: No sound
+* `"default_place_node"`: Play e.g. `default_place_node.ogg`
+* `{name = "default_place_node"}`: Same as above
+* `{name = "default_place_node", gain = 0.5}`: 50% volume
+* `{name = "default_place_node", gain = 0.9, pitch = 1.1}`: 90% volume, 110% pitch
 
 Special sound files
 -------------------
 
 These sound files are played back by the engine if provided.
 
- * `main_menu`: Looped sound in the main menu (gain = 1.0)
  * `player_damage`: Played when the local player takes damage (gain = 0.5)
  * `player_falling_damage`: Played when the local player takes
    damage by falling (gain = 0.5)
-
+ * `default_dig_<groupname>`: Default node digging sound
+   (see node sound definition for details)
+ * `main_menu`: Looped sound in the main menu (gain = 1.0). This sound is
+   not actually part of the modding API, as it's used by the client. Players
+   can place this sound in `~/.minetest/sounds` to enable main menu music
```

Comment:
```
in `$path_user/sounds`
```
is more generic.

I know players would rather look here than in `menu_lua_api.txt` but that would be the correct location for this sound documentation. `lua_api.txt` is only for server mods.

Reply:
I prefer this is not added here, firstly for the reason above, secondly because it is a messy feature that makes no sense (music should be provided by the game, music is not suitable for a game engine) and this will hopefully be removed once we have game-provided music.